builtin/logical/pki/issuing/cert_verify.go

```diff
@@ -79,18 +53,11 @@ func VerifyCertificate(ctx context.Context, storage logical.Storage, issuerId Is
 		DisablePathLenChecks:           false,
 		DisableNameConstraintChecks:    false,
 	}
-
 	if err := entSetCertVerifyOptions(ctx, storage, issuerId, &options); err != nil {
 		return err
 	}
 
-	certificate, err := convertCertificate(parsedBundle.CertificateBytes)
-	if err != nil {
-		return err
-	}
-
-	_, err = certificate.Verify(options)
-	return err
+	return certutil.VerifyCertificate(parsedBundle, options)
 }
 
 func convertCertificate(certBytes []byte) (*ctx509.Certificate, error) {
```

Comment:
`convertCertificate` can be removed now, since it was moved to the certutil package.

Reply:
Gotcha, removed.